src/mom.jl

```diff
@@ -319,3 +319,39 @@ function smi(hlc::Matrix{T}; n::Int64=13, nFast::Int64=2, nSlow::Int64=25, nSig:
     out[:,2] = maSig(out[:,1], n=nSig)
     return out
 end
+
+
+"""
+MFI (Money Flow Index) gives almost the same results as tradingview.com
```

Comment:
Documentation comments should follow the same convention as other functions (function signature in triple-backticks followed by indicator name/interpretation two lines below).